Project.toml

```
@@ -24,7 +24,7 @@ DataStructures = "0.18.3"
 Distributions = "0.25"
 MLJModelInterface = "1.6"
 SpecialFunctions = "0.8, 0.9, 0.10, 1, 2"
-StatsBase = "0.33"
+StatsBase = "0.32, 0.33, 0.34"
```

I don't think we want to add support for 0.32?! Not sure how this ended up in the PR:
```suggestion
StatsBase = "0.33, 0.34"
```

---

I agree. Seems @yebai added this in 7120ebe .